### ISSUE TITLE
Unable to delete jail

### ISSUE THREAD

**Issue by djlawrence4** [graywolf336/Jail#107]
I tried the force command to force the release of prisoners, and then when I go to delete the jail it says unsuccessful.

**Comment by graywolf336** [comment]
Turn on debugging and then try again. If it fails again, copy the console output. Use `/togglejaildebug` to enable debugging.

**Comment by graywolf336** [comment]
@djlawrence4 Any update on the items above?

**Comment by djlawrence4** [comment]
I apologize I have been busy recently. I did find the issue. I had to see who was in jail by going into the config file that stores the info of who was in jail. I forced release the individual and then was able to delete the jail but the force all command dosn't completly remove all from the database. Hopefully this helps. Other than that I have been using your jail plugin for a while and I think its amazing and great job scripting all of the cool features. So I thank you and all who worked on this project.
